src/dispatch/static/dispatch/src/search/SearchFilterCreateDialog.vue

```diff
@@ -68,6 +68,18 @@
                           <incident-status-multi-select v-model="filters.status" />
                         </v-list-item-content>
                       </v-list-item>
+                      <v-list-item>
+                        <v-list-item-content>
+                          <v-select
+                            :items="visibilities"
+                            v-model="filters.visibility"
```

Comment:
`visibility` is missing in the `filters` dict in the `data()` function and the `this.$watch` at the end.

Reply:
Done